generators/vsix-vs-win/BotBuilderVSIX-V4/source.extension.vsixmanifest

```diff
@@ -20,6 +20,6 @@
         <Asset Type="Microsoft.VisualStudio.ProjectTemplate" d:Source="File" Path="ProjectTemplates" />
     </Assets>
     <Prerequisites>
-        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,16.0)" DisplayName="Visual Studio core editor" />
+        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,)" DisplayName="Visual Studio core editor" />
```

Comment:
Why have this be different than the [15.0, 17.0) notation above?

Reply:
This should be fixed now.